### PR TITLE
ARXIVNG-2946: incorporate more admin feedback

### PR DESCRIPTION
**Pull request by SBBCornell**
This PR incorporates more admin feedback into submission-ui. It's mostly text changes.

On a related note the ticket 2946 has some discussion items for the team before I move it into the 'done' column.

### CODE REVIEW

File: submit/templates/submit/cross_list.html [hunk 1]
```diff
@@ -11,15 +11,18 @@
...
         </p>
       </li>
     </ol> -->
 
+    <p>You may cross-list your article to another relevant category here. If accepted your article will appears in the regular listing for that category (in the cross-list section).<br><br>
+      Readers consider <strong>excessive or inappropriate cross-listing to be bad etiquette</strong>. It is rarely appropriate to add more than one or two cross-lists. Note you are unlikely to know that a cross-list is appropriate unless you are yourself a reader of the archive which you are considering.</p>
 
     <p class="has-text-weight-bold">Selected cross-list Categories (optional):</p>
     {% for category, subform in formset.items() %}
+    <!-- TODO: limit number of cross links to no more than 5 total -->
```
Comment (mhl10) [inline submit/templates/submit/cross_list.html:25]: FYI, submission-ui currently just [warns if there are > 3](https://github.com/arXiv/arxiv-submission-ui/blob/67c193ed0baf45fdedf00c7fae54a49beee7e428/submit/controllers/ui/new/classification.py#L207).

**Review by mhl10 — APPROVED** [review]
Looks good!

**Review by bdc34 — APPROVED** [review]
Looks good.

**Review by DavidLFielding — COMMENTED** [review]
Looks good. Catching up with reviews...

**Pull request merged**